yb-voyager/cmd/export.go

```diff
@@ -238,11 +241,24 @@ func validateOracleParams() {
 
 }
 
+func validateSourcePassword(cmd *cobra.Command) {
+	if cmd.Flags().Changed("source-db-password") {
+		return
+	}
+	fmt.Print("Password to connect to source:")
+	bytePassword, err := term.ReadPassword(int(syscall.Stdin))
+	if err != nil {
+		utils.ErrExit("read password: %v", err)
+		return
+	}
+	fmt.Print("\n")
+	source.Password = string(bytePassword)
+}
+
```

Comment:
Is this also tested with special characters in password while entering through input?